kubecost-cloud-request-right-sizing.md

```diff
@@ -0,0 +1,26 @@
+# Kubecost Cloud: Request Right-Sizing Recommendations
+
+{% hint style="info" %}
+This documentation should only be consulted when using Kubecost Cloud! For information about request right-sizing recommendations for self-hosted Kubecost, see [here](https://docs.kubecost.com/using-kubecost/navigating-the-kubecost-ui/savings/auto-request-sizing).
+{% endhint %}
+
+Kubecost Cloud is able to provide recommendations for right-sizing your container requests to ensure they are as cost-effective as possible. Recommendations are provided for all namespaces within your cluster.
+
+## Viewing request recommendations
+
+You can access request right-sizing by selecting _Savings_ in the left navigation, then select the _Right-size your container requests_ panel.
+
+On the Request Right-sizing Recommendations page, you will see a table containing all namespaces/controller pairs and the cluster and container associated with each. You will also see the requested and recommended RAM/CPU, the current efficiency, and finally estimated monthly savings by adopting recommendations.
+Configuring your recommendations
+
+The displayed right-sizing recommendations are calculated by taking into account your environment profile. You can optionally configure this for more optimal results by selecting _Customize_ above the table.
+* Window: The range of time Kubecost will read for resource activity to determine its recommendations.
+* Profile: Refers to the type of environment. The selected value for Profile may restrict you from customizing certain other values.
+  * Production: Stable container activity, will provide some extra space for potential spikes in activity.
+  * Development: Container can tolerate small amount of instability, will run somewhat close to capacity.
+  * High availability: Container should avoid instability at all costs, will size container with lots of extra space to account for unexpected spikes in activity.
+* CPU/RAM target utilization: These can be set to limit recommended utilization of resources below a percentage threshold. Currently, the only CPU/RAM recommendation algorithm supported is _Percentile_.
+* CPU/RAM percentile: Percentage of data points that will be sampled within your window range. Outlier data will be filtered out when determining recommendations.
```

Comment:
I think the algorithm mention could have its own bullet point. If not, we could delete the sentence I recommended altogether.

```suggestion
* CPU/RAM recommendation algorithm: The algorithm used to compute the recommendations. Currently, the only supported option is _Percentile_.
* CPU/RAM target utilization: These can be set to limit recommended utilization of resources below a percentage threshold.
* CPU/RAM percentile: Percentage of data points that will be sampled within your window range. Outlier data will be filtered out when determining recommendations.
```

Reply:
Yeah, you're right. My initial thought was that this wasn't configurable, and therefore didn't need to be added, but let me go ahead and fix that